crates/wdk-sys/build.rs

```diff
@@ -135,11 +136,23 @@ fn main() -> anyhow::Result<()> {
         ),
     ];
 
+    let mut handles = Vec::<JoinHandle<Result<(), ConfigError>>>::new();
     for out_path in out_paths {
-        generate_constants(&out_path, config.clone())?;
-        generate_types(&out_path, config.clone())?;
-        generate_ntddk(&out_path, config.clone())?;
-        generate_wdf(&out_path, config.clone())?;
+        let temp_config = config.clone();
+        let handle: JoinHandle<Result<(), ConfigError>> = thread::spawn(move || {
+            generate_constants(&out_path, &temp_config)?;
```

Comment:
Each of these 4 generates could actually be paralllelized

Reply:
D'you know, I think that was my original intent and I got distracted by various compile errors (still learning the nuances of Rust threads.)  I'll make that change and check that everything still works as expected.